doc/lua_api.md

```diff
@@ -3928,6 +3928,28 @@ The following functions provide escape sequences:
     * Removes all color escape sequences.
 
 
+Coordinate System
+=================
+
+Luanti uses a **left-handed** coordinate system: Y is "up", X is "right", Z is "forward".
+This is the convention used by Unity, DirectX and Irrlicht.
+It means that when you're pointing in +Z direction in-game ("forward"), +X is to your right; +Y is up.
+
+Consistently, rotation is **left-handed** as well:
+When your thumb points in the direction of the rotation axis,
+your curled fingers point in the direction the rotation goes.
```

Comment:
I don't think we should explain this here.
Just add a link to wikipedia: https://en.wikipedia.org/w/index.php?title=Right-hand_rule

Reply:
sure. maybe i'll submit a beginner-friendlier version of this to the docs repo.